os/drivers/lcd/mipi_lcd.c

```diff
@@ -144,6 +144,69 @@ static int lcd_putrun(FAR struct lcd_dev_s *dev, fb_coord_t row, fb_coord_t col,
 	return OK;
 }
 
+static int lcd_setarea(fb_coord_t row_start, fb_coord_t row_end, fb_coord_t col_start, fb_coord_t col_end)
+{
+	int transfer_status = 0;
+	struct mipi_dsi_msg msg_row, msg_col, msg;
+	FAR struct mipi_lcd_dev_s *priv = &g_lcdcdev;
+
+	priv->config->lcd_mode_switch(false);
+
+	u8 cmd_addr_0[1] = {0x00};
```

Comment:
In here, you sent 0x29, 0x2B, 0x29, 0x2A, 0x2C. As I tested, sending just 0x2B and 0x2A is also working fine (ie update row and column).
May I know the reason to send 0x29 and 0x2C?